draft-wood-cfrg-aead-limits.md

```diff
@@ -193,7 +211,7 @@ advantage does not exceed the probability of success, provided that
 `v <= (p * 2^106) / 8l`. In turn, this implies that `v <= (p * 2^106) / 8l`
 is the corresponding limit.
 
-# AEAD Limits and Requirements {#limits}
+# Single-User AEAD Limits {#limits}
```

Comment:
These are single-key, not single-user, right?

Reply:
As asked on the email, I don't understand how you're distinguishing between these two. (To me, they're the same.)